mtg_card_identifier/CardScannerFile.swift

```diff
@@ -0,0 +1,125 @@
+//
+//  CardScannerFile.swift
+//  mtg_card_identifier
+//
+//  Created by mac on 8/16/23.
+//
+
+import SwiftUI
+import UIKit
+import VisionKit
+
+struct CardScanner: UIViewControllerRepresentable {
+    
+    //    @Binding var startScanning: Bool
+    @Binding var scanedCard: [String]
+    
+    func makeCoordinator() -> Coordinator {
+        Coordinator(self)
+    }
+    
+    func makeUIViewController(context: Context) -> DataScannerViewController {
+        
+        let vc = DataScannerViewController(
+            recognizedDataTypes: [.text()],
+            qualityLevel: .balanced,
+            recognizesMultipleItems: true,
+            isHighFrameRateTrackingEnabled: true
+            //            isGuidanceEnabled: true,
+//            Don't know if we want that to work but it's basically done
+//            isHighlightingEnabled: true
+        )
+        let partialTransparentView = PartialTransparentView(cutout: CGRect(x: 31, y: 75, width: 350, height: 470))
+        partialTransparentView.translatesAutoresizingMaskIntoConstraints = true
+        partialTransparentView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        partialTransparentView.frame = vc.overlayContainerView.bounds
+        
+        vc.view.addSubview(partialTransparentView)
+        vc.regionOfInterest = CGRect(x: 31, y: 75, width: 350, height: 430)
```

Comment:
I think the CGRect on lines 32 and 38 should be the same, right? The height is different by 40 currently. 
I would personally pull this out into a separate variable for `var cardShape: CGRect` or something.

Reply:
Yeah I left the regionOfInterest like that so it wouldn't scan like the very bottom of the card. However I was unsure if we were going to use that in the api call to find the card haha.